Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN chmod 600 /root/.ssh/id_rsa
 RUN pub get
 RUN pub run dart_dev format --check
 RUN dartanalyzer .
-RUN pub run dependency_validator -i dart_style,pedantic
+RUN pub run dependency_validator -i dart_style,build_runner,build_test,pedantic
```

Comment:
This reminds me.. we should update dep validator to ignore the `build_` packages by default since they are well-known builder packages. Or alternatively, we should check `.dart_tool/build/` for usages

Reply:
https://jira.atl.workiva.net/browse/CPLAT-6965